docs/templates/language-reference.md

```diff
@@ -49,10 +49,30 @@ Depending on the language, a brief description of the recommended way to handle
 If available, include examples for connecting in an app
 (taken from https://examples.docs.platform.sh/).
 
-### Configuration reader
+{{< codetabs >}}
 
-If available, include the configuration reader available for the given language
-(for example, for [Python](https://github.com/platformsh/config-reader-python)).
+---
+title=<EXAMPLE_1>
+file=static/files/fetch/examples/<LANGUAGE_TYPE>/<SERVICE>
+highlight=js
+---
+
+<--->
+
+---
+title=<EXAMPLE_2>
+file=static/files/fetch/examples/<LANGUAGE_TYPE>/<SERVICE>
+highlight=js
+---
+
+{{< /codetabs >}}
+
+<!-- If available, include the configuration reader available for the given language
+(for example, for [Python](https://github.com/platformsh/config-reader-python)). -->
```

Comment:
:lady_beetle: This example worked with the previous wording, but doesn't fit the pattern of how with do examples with variables.

Reply:
So I guess it's safe to remove that part altogether? Since the example should be clear enough.